web-common/src/components/date-picker/Litepicker.svelte

```diff
@@ -8,7 +8,9 @@
     defaultStart: string,
     defaultEnd: string,
     openOnMount = false,
-    editingDate: 0 | 1 = 0;
+    editingDate: 0 | 1 = 0,
+    min: string,
+    max: string;
```

Comment:
Can you explain the purpose of min/max addition here?

Reply:
We have custom range selector in comparison as well. In case there is no comparison, we fallback to the all time range.

**Before**

![image](https://github.com/rilldata/rill/assets/4402679/1b33b4b2-0646-462c-b7fa-f6776d67f9d3)

**After**

![image](https://github.com/rilldata/rill/assets/4402679/e42d621e-2a23-44e5-9798-52737d22ebc5)